config/config.go

```diff
@@ -297,6 +302,21 @@ func configureLogger(verbosity log.Level, w io.Writer) {
 	log.SetDefault(log.New(h))
 }
 
+func logToFilePath(path string) *os.File {
+	targetDir := filepath.Join(path, "imup", "logs")
```

Comment:
if they specify a file path they're gonna expect it to be in the exact path. if it's supposed to be a directory and not a file we should update the lingo, but they'll still not expect creation of subdirs